src/components/AddNew/AddNew.js

```diff
@@ -0,0 +1,150 @@
+import styled from 'styled-components/macro'
+import Upload from '../Upload/Upload'
+import React, { useState } from 'react'
+
+export default function AddNew({ onAddNewCard }) {
+  const [images, setImages] = useState([])
+
+  return (
+    <AddNewContainer onSubmit={handleSubmit}>
+      <Upload onImage={setImages} required="required" />
+      <p>
+        Umso mehr Infos du speicherst, umso einfacher kannst du deine
+        Lieblingsteile später wieder finden:)
+      </p>
+      <label>
+        <h2>Name (optional):</h2>
+        <input
+          placeholder="z.B. Minikleid oder Lieblingskleid"
+          name="nameOfClothing"
+        />
+      </label>
+      <label>
+        <h2>Kaufdatum (optional):</h2>
+        <input type="date" name="date" />
+      </label>
+      <label>
+        <h2>Geschäft (optional):</h2>
+        <input placeholder="Monki" name="store" />
+      </label>
+      <label>
+        <h2>Preis (optional):</h2>
+        <input
+          placeholder="35"
+          name="price"
+          type="number"
+          step="0.01"
+          min="0"
+        />
+      </label>
+      <h2>Kleidungsart:</h2>
+      <ContainerClothingType>
+        <label>
+          Oberteil
+          <input type="radio" name="clothingType" value="Oberteil" />
+        </label>
+        <label>
+          Kleid
+          <input type="radio" name="clothingType" value="Kleid" />
+        </label>
+
+        <label>
+          Rock
+          <input type="radio" name="clothingType" value="Rock" />
+        </label>
+
+        <label>
+          Hose
+          <input type="radio" name="clothingType" value="Hose" />
+        </label>
+
+        <label>
+          Jacke
+          <input type="radio" name="clothingType" value="Jacke" />
+        </label>
+
+        <label>
+          Schuhe
+          <input type="radio" name="clothingType" value="Schuhe" />
+        </label>
+
+        <label>
+          Accessoire
+          <input type="radio" name="clothingType" value="Accessoire" />
+        </label>
+
+        <label>
+          special stuff💫
+          <input type="radio" name="clothingType" value="special" />
+        </label>
```

Comment:
shouldn't be easier to map through the types of clothes? instead of repeating the code through 30 lines?